cli/src/cluster_query.rs

```diff
@@ -451,7 +451,7 @@ impl ClusterQuerySubCommands for App<'_, '_> {
         )
         .subcommand(
             SubCommand::with_name("rent")
-                .about("Calculate per-epoch and rent-exempt-minimum values for a given account data length.")
+                .about("Calculate per-epoch and rent-exempt-minimum values for a given account data length (account overhead is already included).")
```

Comment:
```suggestion
                .about("Calculate per-epoch and rent-exempt-minimum values for a given account data field length.")
```

Would this be sufficient?  The parenthetic is a bit verbose IMO

Reply:
Done

Reply:
Cool.  Same for the other one?